rollup.config.js

```diff
@@ -0,0 +1,70 @@
+import nodeResolve from 'rollup-plugin-node-resolve'
+import replace from 'rollup-plugin-replace'
+import commonjs from 'rollup-plugin-commonjs'
+import inject from 'rollup-plugin-inject'
+import babel from 'rollup-plugin-babel'
+import json from 'rollup-plugin-json'
+import flow from 'rollup-plugin-flow'
+import uglify from 'rollup-plugin-uglify'
+
+const processShim = '\0process-shim'
+
+const prod = process.env.PRODUCTION
+const mode = prod ? 'production' : 'development'
+
+console.error(`creating ${mode} bundle...`)
```

Comment:
Just because that prints to `stderr` (i.e. metadata) rather than `stdout` (i.e. data). No actual difference, just force of habit! Can change to `console.log` (or remove altogether) if you prefer